obspy/core/inventory/response.py

```diff
@@ -1211,6 +1213,373 @@ def get_evalresp_response(self, t_samp, nfft, output="VEL",
             freqs, output=output, start_stage=start_stage, end_stage=end_stage)
         return response, freqs
 
+    def get_evalresp_norm_resp(self, output, start_stage=None, end_stage=None):
```

Comment:
There are similar functions for reading a pair of dl, sensor resps (nrl style) or a single resp(rdseed -R) on a dataless seed volume. I The combining code should be factored out to remove the duplication and perhaps put in nrl module.